run

```diff
@@ -144,6 +144,11 @@ rsync_back() {
     fi
 }
 
+# set GOPROXY
+GOPROXY=https://proxy.golang.org
```

Comment:
With go 1.13 we no longer need to explicitly set this, since it defaults to `https://proxy.golang.org,direct`

From https://golang.org/doc/go1.13#modules:

> The GOPROXY environment variable may now be set to a comma-separated list of proxy URLs or the special token direct, and its default value is now https://proxy.golang.org,direct. When resolving a package path to its containing module, the go command will try all candidate module paths on each proxy in the list in succession. An unreachable proxy or HTTP status code other than 404 or 410 terminates the search without consulting the remaining proxies.

IIUC, with go 1.13 default, we may even not need `GOPRIVATE` since after `proxy.golang.org fails`, it will try `direct`

Reply:
Ok, I've removed your commit from this PR that included the GOPROXY directive.

Reply:
What about `GOPRIVATE` ? may be don't need it as well?

Reply:
Proxy would have to try, then fail, this seems cleaner. I don't know of any public upbound repos at the moment for code dependencies so this seems more appropriate to go direct and skip the proxy in this case.